CHANGELOG.md

```diff
@@ -25,6 +25,9 @@
 
 ### Changed
 
+* Fixed bindings and comments for `Atomics.wait`.
+  [#3509](https://github.com/rustwasm/wasm-bindgen/pull/3509)
```

Comment:
Could you put this at the bottom?

Reply:
Added it to the fixed section (which I also added as it seemed more appropriate than putting it under changed).

Reply:
Good thinking.